front/components/spaces/ConfirmDeleteSpaceDialog.tsx

```diff
@@ -54,7 +51,21 @@ export function ConfirmDeleteSpaceDialog({
           </div>
         ) : (
           <>
-            <DialogContainer>{message}</DialogContainer>
+            <DialogContainer className="space-y-4">
+              {hasAgents && (
+                <ContentMessage
+                  variant="warning"
+                  // TODO: change to show names of public agents and then number of unpublished agents
+                  title={`This will break ${dataSourceUsage} existing agent(s)`}
+                />
+              )}
+              <div>
+                <p className="text-sm text-muted-foreground">
+                  Are you sure you want to permanently delete space {spaceName}
```

Comment:
```suggestion
                  Are you sure you want to permanently delete space {spaceName}?
```

Let's add `?` 

<img width="467" alt="Screenshot 2025-06-30 at 10 33 26" src="https://github.com/user-attachments/assets/b20fe2dc-9336-4e3a-8e01-2b9e6e60a078" />
 
<img width="434" alt="Screenshot 2025-06-30 at 10 31 52" src="https://github.com/user-attachments/assets/2fadb8a6-c0fe-4890-962e-22f334ba3eb4" />